app/src/main/java/mozilla/lockbox/extensions/ViewNode+.kt

```diff
@@ -12,7 +12,11 @@ fun AssistStructure.ViewNode.dump(): String {
 
 @TargetApi(Build.VERSION_CODES.O)
 private fun AssistStructure.ViewNode.dumpNode(sb: StringBuilder = StringBuilder()): StringBuilder {
-    val name = htmlInfo?.tag ?: className.split('.').last()
+    val name = if (className.isNullOrBlank()) {
+        htmlInfo?.tag ?: ""
+    } else {
+        htmlInfo?.tag ?: className.split('.').last()
+    }
```

Comment:
Nit: use optionals and the Elvis operator, and sensible descriptive default:

```kotlin
val name = htmlInfo?.tag ?: className?.split('.')?.last() ?: "unknown"
```

Reply:
Perfect